schema.graphql

```diff
@@ -32,20 +33,36 @@ type ItemKitchen{
     status: String!
     orderItemInfo: OrderItemInfo!
 }
-input UpdateItemInput{
-    name: String
-    price: Int
-    img: String
-    description: String
-    type: String
-    tags: [String]
-    rate_ave: Float
-}
+# there is a same input below
```

Comment:
you can delete it then)) i mean delete this commented code

Reply:
Sure, I will do that